static/app/views/performance/newTraceDetails/traceApi/useTraceMeta.tsx

```diff
@@ -145,7 +145,7 @@ export function useTraceMeta(replayTraces: ReplayTrace[]): TraceMetaQueryResults
     Error
   >({
     // eslint-disable-next-line @tanstack/query/exhaustive-deps
-    queryKey: ['traceData', replayTraces],
+    queryKey: ['traceData', replayTraces.map(trace => trace.traceSlug)],
```

Comment:
replayTraces here has slug and timestamp. The timestamp differed slightly between the issue details view and this view. Is it important enough to refetch all this?